core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/runtimeModels.scala

```diff
@@ -276,14 +276,26 @@ object RuntimeUI {
 }
 
 /** Default runtime labels */
-case class DefaultLabels(clusterName: RuntimeName,
+case class DefaultLabels(runtimeName: RuntimeName,
                          googleProject: GoogleProject,
                          creator: WorkbenchEmail,
                          clusterServiceAccount: Option[WorkbenchEmail],
                          notebookServiceAccount: Option[WorkbenchEmail],
                          notebookUserScript: Option[UserScriptPath],
                          notebookStartUserScript: Option[UserScriptPath],
-                         tool: Option[RuntimeImageType])
+                         tool: Option[RuntimeImageType]) {
+  def toMap: LabelMap =
+    Map(
+      "runtimeName" -> runtimeName.asString, //TODO: is this label renaming okay?
```

Comment:
I think we should keep the old key (maybe we can include both `clusterName` and `runtimeName`). Terra UI and AoU both make use of labels.